sample-apps/spark/src/main/java/com/amazon/sampleapp/MetricEmitter.java

```diff
@@ -31,22 +42,63 @@
 
   static String API_COUNTER_METRIC = "apiBytesSent";
   static String API_LATENCY_METRIC = "latency";
+  static String API_SUM_METRIC = "totalApiBytesSent";
+  static String API_LAST_LATENCY_METRIC = "lastLatency";
+  static String API_UP_DOWN_COUNTER_METRIC = "queueSizeChange";
+  static String API_UP_DOWN_SUM_METRIC = "actualQueueSize";
 
   LongCounter apiBytesSentCounter;
   LongValueRecorder apiLatencyRecorder;
+  LongSumObserver totalBytesSentObserver;
+  LongValueObserver apiLastLatencyObserver;
+  LongUpDownCounter queueSizeCounter;
+  LongUpDownSumObserver actualQueueSizeObserver;
+
+  long totalBytesSent;
+  long apiLastLatency;
+  long actualQueueSize;
+
+  //The below API name and status code dimensions are currently shared by all metrics observer in this class.
+  String apiNameValue = "";
+  String statusCodeValue = "";
 
   public MetricEmitter() {
+    String otelExporterOtlpEndpoint = System.getenv("OTEL_EXPORTER_OTLP_ENDPOINT") != null ? System.getenv("OTEL_EXPORTER_OTLP_ENDPOINT") : "0.0.0.0:55680";
+    MetricExporter metricExporter =
+            OtlpGrpcMetricExporter.builder()
+                    .setChannel(
+                            ManagedChannelBuilder.forTarget(otelExporterOtlpEndpoint).usePlaintext().build())
+                    .build();
+
+    IntervalMetricReader.builder()
+            .setMetricProducers(
+                    Collections.singleton(OpenTelemetrySdk.getGlobalMeterProvider().getMetricProducer()))
+            .setExportIntervalMillis(1000)
+            .setMetricExporter(metricExporter)
+            .build();
```

Comment:
why we don't use auto-instr-agent to send metrics anymore?

Reply:
I added this when I tested it locally without java-agent and docker. Will remove this part.